### ISSUE TITLE
Training Problem

### ISSUE THREAD

**Issue by Strive21** [Francis-Rings/StableAnimator#67]
Thank you for your great work！I tried to train your model with my own dataset (several thousand videos). Except changing the resolution to 512*768, other settings were the same. I trained it on a 4-card A800 for about 20 epochs, but the results were average. I would like to know why. I hope you can give me an answer. #
![animation_video](https://github.com/user-attachments/assets/e80740b8-1c01-40cc-bd83-471d97626a05)
![pose_video](https://github.com/user-attachments/assets/23a0fb54-0ad8-4d95-8e68-2883588de459)

**Comment by Francis-Rings** [comment]
Hi, please check the quality and the diversity of your own training videos. It is worth noting that the backgrounds of the selected training videos should remain static, as this helps the diffusion model calculate accurate reconstruction loss. Moreover, the optimal number of epochs for StableAnimator to achieve peak performance is influenced by the quality and scale of the training videos. Since I have observed that PoseNet fails to effectively introduce motion guidance to StableAnimator, I recommend training StableAnimator for more epochs.

**Comment by Strive21** [comment]
> Hi, please check the quality and the diversity of your own training videos. It is worth noting that the backgrounds of the selected training videos should remain static, as this helps the diffusion model calculate accurate reconstruction loss. Moreover, the optimal number of epochs for StableAnimator to achieve peak performance is influenced by the quality and scale of the training videos. Since I have observed that PoseNet fails to effectively introduce motion guidance to StableAnimator, I recommend training StableAnimator for more epochs.
I noticed that the training of unet in the code only has attention. What is the difference between it and full finetune?

**Comment by Strive21** [comment]
> Hi, please check the quality and the diversity of your own training videos. It is worth noting that the backgrounds of the selected training videos should remain static, as this helps the diffusion model calculate accurate reconstruction loss. Moreover, the optimal number of epochs for StableAnimator to achieve peak performance is influenced by the quality and scale of the training videos. Since I have observed that PoseNet fails to effectively introduce motion guidance to StableAnimator, I recommend training StableAnimator for more epochs.

I noticed that the training of unet in the code only has attention. What is the difference between it and full finetune?

**Comment by Francis-Rings** [comment]
> > Hi, please check the quality and the diversity of your own training videos. It is worth noting that the backgrounds of the selected training videos should remain static, as this helps the diffusion model calculate accurate reconstruction loss. Moreover, the optimal number of epochs for StableAnimator to achieve peak performance is influenced by the quality and scale of the training videos. Since I have observed that PoseNet fails to effectively introduce motion guidance to StableAnimator, I recommend training StableAnimator for more epochs.
> 
> I noticed that the training of unet in the code only has attention. What is the difference between it and full finetune?

Training the entire SVD is particularly computationally intensive, with GPU memory consumption exceeding 80GB. Training the attention modules of SVD is a practical trade-off strategy.

**Comment by Strive21** [comment]
> > > Hi, please check the quality and the diversity of your own training videos. It is worth noting that the backgrounds of the selected training videos should remain static, as this helps the diffusion model calculate accurate reconstruction loss. Moreover, the optimal number of epochs for StableAnimator to achieve peak performance is influenced by the quality and scale of the training videos. Since I have observed that PoseNet fails to effectively introduce motion guidance to StableAnimator, I recommend training StableAnimator for more epochs.
> > 
> > 
> > I noticed that the training of unet in the code only has attention. What is the difference between it and full finetune?
> 
> Training the entire SVD is particularly computationally intensive, with GPU memory consumption exceeding 80GB. Training the attention modules of SVD is a practical trade-off strategy.

OK，thanks！
